app/assets/javascripts/config/angularRoutes.js.coffee

```diff
@@ -1130,7 +1130,7 @@
         ]
     })
 
-    $urlRouterProvider.otherwise('/') # default to welcome screen
+    $urlRouterProvider.when('/listings', '/').otherwise('/') # default to welcome screen
```

Comment:
yeah, I think I concentrated too much on 'how to redirect' that I didn't actually take time to think what that `otherwise` mean and do. Will remove `when`